README.md

```diff
@@ -21,3 +21,23 @@ Typically the expression will be a `begin-end` block:
         # lots of compute here...
     end
 ```
+
+#### Preferences support
+
+Tracepoints can be toggled or filtered module-wide with a `LocalPreferences.toml` file:
+
+```toml
+[MyPackage.Tracy]
+enabled = true
+whitelist = ['foo', 'bar.*']
```

Comment:
Would be good to describe what the glob pattern means here? Maybe some explicit examples?

Reply:
Should also document somewhere how the dynamic enable feature interact with the preference one. Which one take precedence etc.

Reply:
> Would be good to describe what the glob pattern means here? Maybe some explicit examples?
Yeah, I should document that this is parsed as a regex always